hq/app/aws/ec2/EC2.scala

```diff
@@ -61,12 +63,19 @@ object EC2 {
     } yield sortSecurityGroupsByInUse(flaggedSgsWithVpc, sgUsage)
   }
 
+  def refreshSGSReports(accounts: List[AwsAccount])(implicit ec: ExecutionContext): Attempt[List[Either[FailedAttempt, RefreshTrustedAdvisorCheckResult]]] = {
+    Attempt.traverseWithFailures(accounts) { account =>
```

Comment:
Is there any value keeping all the failures if they're only going to be thrown away?

See [Line 30](https://github.com/guardian/security-hq/pull/31/files#diff-59b0ca98fa263a392fac7315f4bf946bR30) and [Line 41](https://github.com/guardian/security-hq/pull/31/files#diff-59b0ca98fa263a392fac7315f4bf946bR41) of the Controller.

It looks like this functionality would fail silently - there isn't even any logging I can see to let us know if there are problems with the refreshing.

Reply:
I'm getting a `Unknown error while making an API call to AWS' AmazonEC2 service` on some areas of the dashboard

Whereas the application is logging:
```
Failed to execute AWS SDK operation
com.amazonaws.services.ec2.model.AmazonEC2Exception: You are not authorized to perform this operation
```

Might be helpful to have a more detailed error o n the dashboard?

Reply:
@adamnfish regarding failing silently, I can make changes so that it won't retrieve SGS for the failing accounts, or simply can log the accounts failing refresh.